src/components/extensive/WorkdayCollapseGrid/hooks.ts

```diff
@@ -14,23 +62,35 @@ export interface SectionRow {
 }
 
 export function calculateTotals(demographics: Demographic[]) {
-  const counts = demographics.reduce(
-    function (counts, { type, amount }) {
-      counts[type] += amount;
-      return counts;
-    },
-    { gender: 0, age: 0, ethnicity: 0 }
-  );
+  const { framework } = useFrameworkContext();
+  const initialCounts = getInitialCounts(framework);
+  const counts = demographics.reduce(function (counts, { type, amount }) {
+    const typedType = type as keyof FrameworkDemographicCountTypes<typeof framework>;
+    counts[typedType] += amount;
+    return counts;
+  }, initialCounts);
+
+  let total: number = 0;
+  let countsMatch: boolean = false;
+
+  if (counts) {
+    if (isHBFDemographicCounts(counts, framework)) {
+      total = Math.max(counts.age, counts.gender, counts.caste);
+      countsMatch = uniq([counts.age, counts.gender, counts.caste]).length === 1;
```

Comment:
I think this should just be `true` or maybe `counts.gender > 0`. The status in `useTableStatus` looks like it'll never by "complete" for HBF. I think I'd make these changes:
* Rename `countsMatch` to `complete` to make it a bit more generic, and set it to `counts.gender > 0` in this block
* Set `total` in this block to `counts.gender` and then the change to the return value of `useTableStatus` below can be reverted.